src/Brahma.FSharp.OpenCL.Translator/Body.fs

```diff
@@ -126,7 +126,11 @@ module rec Body =
             | _ -> return! Type.translate var.Type
         }
 
-        return VarDecl(varType, newName, Some body)
+        let varDecl = VarDecl(varType, newName, Some body)
+        if newName.Contains "Ref" then
```

Comment:
Can we use a more safe way to detect ref variables? What if user created variable with name "Ref"?